script/vsts/nightly-release.yml

```diff
@@ -0,0 +1,39 @@
+name: 1.29.0-nightly$(Rev:r)
```

Comment:
Need to find a way to pull this version number from package.json on master so that it doesn't need to be updated manually each time we roll the railcars